cfgov/ask_cfpb/models/django.py

```diff
@@ -376,7 +377,8 @@ class Answer(models.Model):
         help_text=(
             'Optionally select a custom image to appear when users share this '
             'page on social media websites. If no image is selected, this '
-            'page\'s category image will be used. Minimum size: 1200w x 630h.'
+            'page\'s category image will be used. Minimum size: 1200w x 630h. '
+            'Maximum size: 4096w x 4096h.'
```

Comment:
I'd like the help text to state that 1200x630 is the ideal size, too. Perhaps:

"Minimum (and ideal) size: 1200w x 630h."

Reply:
I endorse this suggestion, though I have a minor quibble with calling it "ideal," something about it feels off to me.  Maybe "Minimum (and recommended) size: 1200w x 620h"

I can't quite articulate what bugs me about "ideal" though, so, could be a personal problem.

Reply:
This reminds me that there was some conflation around minimum size and target size in the related issue.  Do we want to consider this a minimum at all then?  I was just trying to apply validation that's consistent with the help text, but if it's confusing I can remove the validation on minimum size, remove the help text on minimum size, and just use language around suggesting ideal/recommended size.